src/saltext/azurerm/utils/azurerm.py

```diff
@@ -174,7 +178,6 @@ def get_client(client_type, **kwargs):
             base_url=cloud_env.endpoints.resource_manager,
             user_agent_policy=user_agent,
         )
-
```

Comment:
**suggestion (non-blocking):** unless a `pre-commit` check like `black` is adding/removing empty lines, it's best to refrain from unrelated whitespace changes in PRs.